### PR TITLE
avoid url error exception

### PR DESCRIPTION
**Pull request by kouml**
## Summary
To Avoid falling into url error exception, I changed the order of exceptions.

**Reference**
https://github.com/keras-team/keras/pull/10887

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 31 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by olicht — APPROVED** [review]

**Comment by kouml** [comment]
@protoget  @olicht 
What is the current status?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by protoget — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
